Auth/LearningHub.Nhs.Auth/appsettings.json

```diff
@@ -19,8 +19,8 @@
   },
```

Comment:
These changes to the appsettings.json file are not requried (Critical should also be a temporary setting and not hardcoded here).
The release pipeline overrides the setting here anyway (setting the level to Error)

Reply:
@DRJTechnology  This has been reverted